x-pack/solutions/observability/plugins/slo/server/assets/index_templates/sli_index_template.ts

```diff
@@ -13,11 +14,11 @@ import {
   SLO_RESOURCES_VERSION,
 } from '../../../common/constants';
 
-export const SLI_INDEX_TEMPLATE = {
+export const SLI_INDEX_TEMPLATE: IndicesPutIndexTemplateRequest = {
   name: SLI_INDEX_TEMPLATE_NAME,
   index_patterns: [SLI_INDEX_TEMPLATE_PATTERN],
   composed_of: [SLI_COMPONENT_TEMPLATE_MAPPINGS_NAME, SLI_COMPONENT_TEMPLATE_SETTINGS_NAME],
-  priority: 500,
+  priority: 600,
```

Comment:
To override the previous index template matching on broader index pattern, e.g. `.slo-observability.sli-*` instead of `.slo-observability.sli-v3.6*` like now